examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -100,7 +104,7 @@ export interface ITaskListEvents extends IEvent {
 	/**
 	 * Emitted when a task is added/removed respectively.
 	 */
-	(event: "taskAdded" | "taskDeleted", listener: (task: ITask) => void);
+	(event: "taskAdded" | "taskDeleted" | "taskChanged", listener: (task: ITask) => void);
```

Comment:
I would recommend splitting this into 3 separate call signatures so we can document them independently.